ansible/modules/hashivault/hashivault_write.py

```diff
@@ -133,7 +133,7 @@ def hashivault_write(module):
                     returned_data = client.secrets.kv.v2.create_or_update_secret(mount_point=mount_point, cas=cas,
                                                                                  path=secret, secret=write_data)
                 else:
-                    returned_data = client.write(secret_path, **write_data)
+                    returned_data = client.write_data(secret_path, **write_data)
```

Comment:
Was this required for hvac 2?

Reply:
Yes it's a breaking change of 2.0.0 : https://github.com/hvac/hvac/blob/main/CHANGELOG.md#200